clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -22,6 +22,9 @@ def err_drv_unsupported_option_argument : Error<
   "unsupported argument '%1' to option '%0'">;
 def err_drv_unsupported_option_argument_for_target : Error<
   "unsupported argument '%1' to option '%0' for target '%2'">;
+def err_drv_unsupported_opt_removed : Error<
```

Comment:
Is there a test case added for this diagnostic?

Reply:
Yes - here:  https://github.com/intel/llvm/blob/8bfe08d0d4e0438125292e6cfb1232384a6a47ef/clang/test/Driver/sycl-unsupported.cpp#L59